net/qpid-proton/Portfile

```diff
@@ -36,7 +37,11 @@ configure.args-append \
                     -DBUILD_PYTHON=OFF \
                     -DBUILD_GO=OFF \
                     -DBUILD_RUBY=OFF \
-                    -DENABLE_VALGRIND=OFF
+                    -DENABLE_VALGRIND=OFF \
+                    -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
Instead of adding `-DCMAKE_BUILD_TYPE=RelWithDebInfo` to `configure.args` manually, please use
```
cmake.build_type    RelWithDebInfo
```

Reply:
You have not done this yet.